main.py

```diff
@@ -1,10 +1,49 @@
+import json
+
 import init_django_orm  # noqa: F401
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    try:
+        with open("players.json", "r") as file:
+            players_data = json.load(file)
+    except FileNotFoundError:
+        print("players.json file not found.")
+        return
+
+    for nickname, player_data in players_data.items():
+        race_data = player_data["race"]
+        race, _ = Race.objects.get_or_create(
+            name=race_data["name"], defaults={
+                "description": race_data.get("description", "")
+            }
+        )
+
+        for skill_data in race_data.get("skills", []):
+            Skill.objects.get_or_create(
+                name=skill_data["name"],
+                race=race,
+                defaults={"bonus": skill_data["bonus"]}
```

Comment:
Ensure that the 'bonus' field in Skill is correctly handled if it is expected to be a numeric value. If so, consider converting it to the appropriate data type before saving.